packages/common/src/lib/format-date.js

```diff
@@ -0,0 +1,64 @@
+const { formatInTimeZone, zonedTimeToUtc, utcToZonedTime } = require('date-fns-tz');
+const { isValid } = require('date-fns');
+
+const ukTime = 'Europe/London';
```

Comment:
If I'm picky (which I usually am...), a better name would be `ukTimezone`

Reply:
done